### DISCUSSION TITLE
Histogram of grain size distribution

### DISCUSSION THREAD

**Discussion by sajutabraham** [neperfepx/neper#267] [Q&A] [answered]
@rquey Could you please describe how to plot a grain size histogram once the tesselation of mean '**D**' and the standard deviation '**d**' is made using neper? Extraction of the volume or diameter of the grains from the .tess file is required for the same. Your help is much appreciated.

**Comment by rquey** [comment, marked as answer]
Use `-statcell diameq` to get the grain values, and then use your favorite tool to compute / plot the corresponding distribution.

**Reply by sajutabraham** [reply]
Thank you very much!
I have one more doubt. How to obtain the values of internal grains (avoiding the surface grains) alone?

**Reply by rquey** [reply]
`-statcell diameq,body` and then use only body > 0 values.
